### PR TITLE
add domain to the WmiClientWrapper constructor

### PR DESCRIPTION
**Pull request by ArminNaCl**
Dear FTP solutions,

Since in my case, I had to specify the domain of Windows Server when I send a query. I add this to the WmiClientWrapper constructor.

would you please check it and merge it, so other users can use it too?

Regard
NaCl

### CODE REVIEW

**Comment by initialed85** [comment]
Hey @ArminNaCl thx for submitting a PR!

I'll take a look shortly and if it's all good I'll get it merged in and do a new release to PyPI.

File: wmi_client_wrapper/wrapper.py [hunk 1]
```diff
@@ -56,9 +57,10 @@ def _make_credential_args(self):
 
         # the format is user%pass
         # NOTE: this is an injection vulnerability
-        userpass = "--user={username}%{password}".format(
+        userpass = "--user={domain}{username}%{password}".format(
             username=self.username,
             password=self.password,
+            domain = str(self.domain) + "/" if self.domain else ""
```
Comment (initialed85) [inline wmi_client_wrapper/wrapper.py:63]: image

Just as an FYI.

I'll run https://github.com/psf/black over the master before I deploy the new release to PyPI so it's all good.

**Review by initialed85 — APPROVED** [review]
LGTM- I'll merge this in and do a release to PyPI shortly

**Comment by ArminNaCl** [comment]
Thanks for your support

**Pull request merged**